packages/cli/src/lib/snapshot/snapshotCommon.ts

```diff
@@ -79,14 +79,14 @@ export async function getTargetOrg(config: Config, target?: string) {
 
 export function handleSnapshotError(err?: Error) {
   if (err instanceof SnapshotOperationTimeoutError) {
-    cli.info(operationGettingTooMuchTimeMessage(err.snapshot));
```

Comment:
So did you find what's the differences between `info` and `log`?

Reply:
yeah. `log` is just calling `info`.
I also did that switch since we are using `cli.log` everywhere else in the code.

Reply:
Ok so switching for uniformity 👍 all's well